src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/HealthCertifiedPerson/HealthCertifiedPersonCellModel.swift

```diff
@@ -62,6 +62,19 @@ class HealthCertifiedPersonCellModel {
 		}
 
 		self.onTapToDelete = nil
+		
+		func shouldShowCaptionFor(initialCertificate certificate: HealthCertificate) -> Bool {
+			// Test certificates that are invalid or blocked.
+			if certificate.type == .test {
```

Comment:
Would it make sense to cater both conditions with one if statement?

Reply:
That's a good question. I had that in your way, but then the code are harder to read.